environment.yml

```diff
@@ -1,9 +1,567 @@
+    - pyasn1==0.4.8
+    - pyasn1-modules==0.2.8
+    - pycocotools==2.0.0
+    - pycodestyle==2.6.0
+    - pycparser==2.20
+    - pydocstyle==6.0.0
+    - pyflakes==2.2.0
+    - pyglet==1.5.0
+    - pyjwt==2.1.0
+    - pylint==2.8.2
+    - pynacl==1.4.0
+    - pynvml==8.0.4
+    - pyopenssl==20.0.1
+    - pyproj==3.0.1
+    - pyrsistent==0.17.3
+    - pyspark==3.1.1
+    - python-editor==1.0.4
+    - python-jsonrpc-server==0.4.0
+    - python-language-server==0.35.0
+    - pytorch-transformers==1.0.0
+    - pytz==2021.1
+    - pywavelets==1.1.1
+    - pyyaml==5.4.1
+    - pyzmq==22.0.3
+    - qtconsole==5.1.0
+    - qtpy==1.9.0
+    - quantlib==1.22
+    - querystring-parser==1.2.4
+    - rai-core-flask==0.2.0
+    - raiwidgets==0.2.2
+    - rasterio==1.1.5
+    - ray==1.3.0
+    - redis==3.5.3
+    - regex==2021.4.4
+    - requests==2.25.1
+    - requests-oauthlib==1.3.0
+    - rope==0.19.0
+    - rsa==4.7.2
+    - ruamel-yaml==0.17.4
+    - ruamel-yaml-clib==0.2.2
+    - s3transfer==0.3.7
+    - sacremoses==0.0.45
+    - scikit-image==0.17.2
+    - scikit-learn==0.22.2.post1
+    - scipy==1.5.2
+    - scp==0.13.3
+    - scrapbook==0.5.0
+    - seaborn==0.11.1
+    - secretstorage==3.3.1
+    - semver==2.13.0
+    - send2trash==1.5.0
+    - sentencepiece==0.1.95
+    - setuptools==50.3.0
+    - shap==0.34.0
+    - shapely==1.7.0
+    - simpervisor==0.4
+    - six==1.16.0
+    - skl2onnx==1.4.9
+    - sklearn==0.0
+    - sklearn-pandas==1.7.0
+    - smart-open==1.9.0
+    - smmap==4.0.0
+    - sniffio==1.2.0
+    - snowballstemmer==2.1.0
+    - snuggs==1.4.7
+    - soupsieve==2.2.1
+    - spacy==2.1.8
+    - sqlalchemy==1.4.14
+    - sqlparse==0.4.1
+    - srsly==1.0.5
+    - sshtunnel==0.1.5
+    - statsmodels==0.12.2
+    - tabulate==0.8.9
+    - tenacity==7.0.0
+    - tensorboard==2.1.1
+    - tensorboard-data-server==0.6.1
+    - tensorboard-plugin-wit==1.8.0
+    - tensorboardx==2.2
+    - tensorflow==2.1.0
+    - tensorflow-estimator==2.1.0
+    - tensorflow-gpu==2.1.0
+    - termcolor==1.1.0
+    - terminado==0.9.4
+    - terminaltables==3.1.0
+    - testpath==0.4.4
+    - textwrap3==0.9.2
+    - thinc==7.0.8
+    - tifffile==2020.9.3
+    - timezonefinder==5.2.0
+    - tokenizers==0.9.4
+    - toml==0.10.2
+    - torch==1.8.1
+    - torch-tb-profiler==0.1.0
+    - torchvision==0.9.1
+    - tornado==6.1
+    - tqdm==4.60.0
+    - traitlets==4.3.3
+    - transformers==4.1.0
+    - typed-ast==1.4.3
+    - typing-extensions==3.10.0.0
+    - ujson==4.0.2
+    - urllib3==1.25.11
+    - uuid==1.30
+    - vsts==0.1.25
+    - vsts-cd-manager==1.0.2
+    - waitress==2.0.0
+    - wasabi==0.8.2
+    - webencodings==0.5.1
+    - websocket-client==0.59.0
+    - websockets==9.0.2
+    - werkzeug==1.0.1
+    - widgetsnbextension==3.5.1
+    - wrapt==1.12.1
+    - xarray==0.16.2
+    - xgboost==0.90
+    - xmltodict==0.12.0
+    - yapf==0.31.0
+    - yarl==1.6.3
+    - zipp==3.4.1
+    - zope-event==4.5.0
+    - zope-interface==5.4.0
```

Comment:
How much time does it take to install these libs explicitly ?

Reply:
Those are preinstalled libraries. It came because we exported existing conda environment (AML). In the upcoming PR (local installation changes), I will generate a fresh Yml file with only the required dependencies for local installation